cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -785,6 +785,15 @@
     public static final String DISCOVERY_TIMEOUT_SECONDS = "monitor.handler.service.discovery.timeout.seconds";
   }
 
+  /**
+   * Runtime Monitor constants.
+   */
+  public static final class RuntimeMonitor {
+    public static final String RUNTIME_MONITOR_POLLTIME_MS = "runtime.monitor.polltime.ms";
```

Comment:
Since the constant is already under `RuntimeMonitor` class, no need to repeat that in the constant name. Just call it `POLL_TIME_MS` is fine (poll time are two words).

Reply:
BTW, for program related configs, usually we have the `app.program.runtime.` prefix.